pkg/apiclient/workflowarchive/workflow-archive.proto

```diff
@@ -29,4 +34,10 @@ service ArchivedWorkflowService {
     rpc DeleteArchivedWorkflow (DeleteArchivedWorkflowRequest) returns (ArchivedWorkflowDeletedResponse) {
         option (google.api.http).delete = "/api/v1/archived-workflows/{uid}";
     }
+    rpc ListArchivedWorkflowLabel (ListArchivedWorkflowLabelRequest) returns (github.com.argoproj.argo_workflows.v3.pkg.apis.workflow.v1alpha1.LabelKeys) {
+        option (google.api.http).get = "/api/v1/archived-workflows-labels/keys";
```

Comment:
I think this should be `ListArchivedWorkflowLabelsKeys`?

`/api/v1/archived-workflows-label-keys`